docs/lib-documentation.md

```diff
@@ -120,17 +120,29 @@ Repartitions the network so that the connections between services match the spec
 * `partitionConnections`: Definitions of the connection state between the new partitions. If a connection between two partitions isn't defined in this map, the default connection will be used. Connections are not directional, so an error will be thrown if the same connection is defined twice (e.g. `Map[A][B] = someConnectionInfo`, and `Map[B][A] = otherConnectionInfo`).
 * `defaultConnection`: The network state between two partitions that will be used if the connection isn't defined in the partition connections map.
 
-### waitForEndpointAvailability(ServiceID serviceId, String httpMethod, uint32 port, String path, String requestBody, uint32 initialDelaySeconds, uint32 retries, uint32 retriesDelayMilliseconds, String bodyText)
-Waits until a service endpoint is available by making requests to the endpoint using the given parameters. An error is thrown if the number of retries is exceeded.
+### waitForEndpointAvailabilityHttpGet(ServiceID serviceId, uint32 port, String path, String requestBody, uint32 initialDelayMilliseconds, uint32 retries, uint32 retriesDelayMilliseconds, String bodyText)
```

Comment:
Same comment - can we rename this to `waitForHttpGetEndpointAvailability`, to read a bit more naturally?